Entities/Character/Character.gd

```diff
@@ -52,27 +57,25 @@ func send_inputs(delta):
 	player_input['inside_circle'] = inside_circle
 	player_input['switch_rotation'] = switch_rotation
 	player_input['shoot'] = shoot
-	print("charaacter input:" + str(player_input['inside_circle']))
 	switch_rotation = false
 	shoot = false
 	if !being_removed:
 		emit_signal("input_ready", player_input)
 
-func got_shot(dmg):
+func got_shot(dmg, killer):
```

Comment:
"killer" is misleading, because it points to a source of damage, it doesn't need to only be used when being killed